packages/utils/odsp-doclib-utils/src/odspErrorUtils.ts

```diff
@@ -191,6 +193,7 @@ export function createOdspNetworkError(
 			}
 			error = new NonRetryableError(
 				errorMessage,
+				// eslint-disable-next-line import/no-deprecated
```

Comment:
Unfortunately, these have to remain as is because in `createOdspNetworkError` the `error` variable is typed as `IFluidErrorBase & OdspError`, and `OdspError` has stronger requirements for `errorType`.